docs/platform/tokyo2-migration/index.md

```diff
@@ -0,0 +1,128 @@
+
+## Benefits of Tokyo 2
+
+The Tokyo 2 data center provides features not available in Tokyo 1:
+
+-  Access to Linode's [Block Storage Service](https://www.linode.com/blockstorage) (available starting Q3 2018).
+-  Access to the [newest Linode plans](https://blog.linode.com/2018/05/17/updated-linode-plans-new-larger-linodes/).
+-  Access to future features on Linode's product roadmap.
+
+## When will My Linodes be Migrated?
+
+You will receive a support ticket from Linode that lists the dates and times of your Tokyo 1 Linodes' migrations. This ticket will be sent to you at least three months in advance of the start of your first migration.
+
+Different Linodes will be scheduled to migrate on different dates and times. The ticket you receive will list all of your Tokyo 1 Linodes' migrations. **Linode will not be able to adjust this schedule of migrations.**
+
+You are able to move your servers to Tokyo 2 before the scheduled migration dates. We recommend that all customers move their servers early. Moving early will help you better control the uptime of your services.
+
+{{< note >}}
+You will receive a credit on your account equal to three months of service for each of your Tokyo 1 Linodes. This credit will cover the cost of creating new servers in Tokyo 2 if you choose to [clone your Linodes](#clone-your-linodes).
+{{< /note >}}
+
+## What are My Options for Migrating?
+
+There are three different options for moving your servers to the new data center. The first two of these methods can be followed before the scheduled migration deadlines for your Linodes.
+
+Regardless of which option you choose, **all of your Tokyo 1 Linodes' IP addresses will change** when moving to the new location. This includes all public and private IPv4 addresses, as well as public and link-local IPv6 addresses.
+
+Later sections in this guide describe how to update your [network interface configuration](#update-your-network-configuration) and [DNS records](#update-dns-records) to use the new IPs.
+
+Please also note that if your Tokyo 1 Linode is enrolled in the [Linode Backup Service](/backups), the saved backups and snapshots that have been created for it **will not move** to the new Tokyo 2 facility. Instead, your Linode will start creating new scheduled backups after it arrives in Tokyo 2 (according to its [backup schedule](/docs/platform/disk-images/linode-backup-service/#schedule-backups)). Your Linode will remain enrolled in the Linode Backup Service.
+
+### Migrate Early
+
+When you receive the support ticket which lists your migration times, you will also see a new banner appear in the dashboard of each of your Tokyo 1 Linodes. This banner will give you the option to initiate your Linode's migration to Tokyo 2 early.
```

Comment:
This banner is a new feature that the dev team is working on in preparation for the Tokyo 2 migration effort